resources/js/components/BaseCard.vue

```diff
@@ -1,41 +1,80 @@
 <template>
-    <div class="card">
-        <img :src="cardImage" class="card__image">
-
-        <a v-if="actionCard" class="card__action" :href="destinationUrl">
-            <div class="">
-                <p class="card__icon">+</p>
-                <p>Novo Evento</p>
-            </div>
-        </a>
-
-        <a v-else class="card__content" :href="destinationUrl">
-            <p class="card__title">
-                {{ title }}
-            </p>
-
-            <p class="card__description">
-                {{ description }}
-            </p>
-
-            <p class="card__date">
-                {{ date }}
-            </p>
-
-            <div class="card__controls">
-                <div class="btn-group">
-                    <a href="#" class="btn btn-outline-primary">
-                        <span>Visualizar</span>
-                    </a>
-                    <a href="#" class="btn btn-outline-success">
-                        <span>Editar</span>
-                    </a>
-                    <a href="#" class="btn btn-outline-danger">
-                        <span>Excluir</span>
-                    </a>
+    <div :class="['card', cardShadowClass]">
+        <div v-if="paperCard" :class="['card--paper', paperCardVariantClass, paperCardPaddingClass]">
+            <slot></slot>
+        </div>
+
+        <div class="card__container" v-else>
+            <img :src="cardImage" :class="{ 'card__image': !actionCard, 'card--action__image': actionCard }">
+
+            <a v-if="actionCard" class="card--action" :href="destinationUrl">
+                <div class="">
+                    <p class="card__icon">
+                        <i class="material-icons">add_circle</i>
+                    </p>
+                    <p>Novo Evento</p>
                 </div>
-            </div>
-        </a>
+            </a>
+
+            <a v-else class="card--content" :href="destinationUrl">
```

Comment:
Acredito que aqui seja `.card__content`.

```suggestion
            <a v-else class="card__content" :href="destinationUrl">
```

Reply:
Aqui seria o mesmo caso do `card--action`, vou resolver tbm.